code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -575,6 +575,67 @@
 	desc = "Used to create condiments and other cooking supplies."
 	condi = 1
 
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/obj/machinery/chem_master/constructable
+	name = "ChemMaster 2999"
+	desc = "Used to seperate chemicals and distribute them in a variety of forms."
+	
+/obj/machinery/chem_master/constructable/New()
+	..()
+	component_parts = list()
+	component_parts += new /obj/item/weapon/circuitboard/chem_master(null)
+	component_parts += new /obj/item/weapon/stock_parts/manipulator(null)
+	component_parts += new /obj/item/weapon/stock_parts/console_screen(null)
+	component_parts += new /obj/item/weapon/reagent_containers/glass/beaker(null)
+	component_parts += new /obj/item/weapon/reagent_containers/glass/beaker(null)
+	
+/obj/machinery/chem_master/attackby(var/obj/item/B as obj, var/mob/user as mob, params)
+
+	if(default_deconstruction_screwdriver(user, "mixer0", "mixer0", B))
+		if(beaker)
+			beaker.loc = src.loc
+			beaker = null
+			reagents.clear_reagents()
+		if(loaded_pill_bottle)
+			loaded_pill_bottle.loc = src.loc
+			loaded_pill_bottle = null
+		return
+		
+	if(exchange_parts(user, B))
+		return
+		
+	if(panel_open)
+		if(istype(B, /obj/item/weapon/crowbar))
+			default_deconstruction_crowbar(B)

```

Comment:
Default_deconstruction_crowbar already checks for item to be crowbar, unless you do something before deconstruction(like ejecting beaker and stuff) you can replace the if check with this